test/corpus/literals.txt

```diff
@@ -11,6 +11,7 @@ foo_bar
 `a "literal"`
 `another
 literal \` foo`
+# Recognized as `_` and `foo` identifiers. Should it be this way?
```

Comment:
IMHO we should parse `_foo` as a single identifier, even though it's technically not a legal R identifier. We could then have a post-processing bit in the diagnostics engine that would check whether things parsed as identifiers are truly "valid" R identifiers.